docs/components/email/email-control-demo.js

```diff
@@ -0,0 +1,91 @@
+
+if (document.getElementById('vue-emailInputDemo')) {
+    new Vue({
+        el: '#vue-emailInputDemo',
+        data: {
+            hasAsterisk: false,
+            hasOptional: false,
+            hasHelpText: false,
+            hasErrorText: false,
+            isDisabled: false,
+            isRequired: false,
+            label: 'Email',
+            helpText: 'abc@abc.com',
+            errorText: 'Please enter valid email',
+            errorTextToDisplay: false,
+            helpTextToDisplay: false,
+            displayTrue: true,
+            displayFalse: false,
+        },
+        computed: {
+            attrDisabled: function () {
+                return (this.isDisabled ? 'disabled' : '');
+            },
+            attrRequired: function () {
+                return (this.isRequired ? 'required' : '');
+            },
+            lblClasses: function () {
+                let classes = [];
+
+                if (this.hasAsterisk) {
+                    classes.push('hxRequired');
+                }
+
+                if (this.hasOptional) {
+                    classes.push('hxOptional');
+                }
+
+                let classNames = classes.join(' ');
+
+                return (classNames === '' ? '' : `class="${classNames}"`);
+            },
+            helpTextClass: function () {
+                let textClasses = [];
+
+                if (this.hasHelpText) {
+                    textClasses.push('hxHelpText');
+                    this.helpTextToDisplay = this.displayTrue;
+                } else {
+                    this.helpTextToDisplay = this.displayFalse;
+                }
+
+                let textClassNames = textClasses.join(' ');
+
+                return (textClassNames === '' ? '' : `class="${textClassNames}"`);
+            },
+            errorTextClass: function () {
+                let errorTextClasses = [];
+                if (this.hasErrorText) {
+                    errorTextClasses.push('hxErrorText');
+                    this.errorTextToDisplay = this.displayTrue;
+                } else {
+                    this.errorTextToDisplay = this.displayFalse;
+                }
+                let errorTextClassNames = errorTextClasses.join(' ');
+
+                return (errorTextClassNames === '' ? '' : `class="${errorTextClassNames}"`);
+            },
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-email-control>
+                      <input
+                        id="emailDemo"
+                        type="email"
+                        ${this.attrDisabled}
+                        ${this.attrRequired}
+                      />
+                      <label
+                        for="emailDemo"
+                        ${this.lblClasses}
+                      >
+                        ${this.label}
+                      </label>
+                      <p ${this.helpTextClass} >${this.helpText}</p>
```

Comment:
1. Please update the logic to toggle on `hidden` attribute.   Keep the `hxHelpText` class.  You will add/remove `hidden`.
1. It will look something like this:

```js
<p class="hxHelpText" ${this.isHelpTextHidden}> ${this.helpText} </p>
```